besu/src/main/java/org/hyperledger/besu/cli/options/unstable/ContractCacheOptions.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.hyperledger.besu.cli.options.unstable;
+
+import org.hyperledger.besu.cli.options.CLIOptions;
+import org.hyperledger.besu.evm.internal.JumpDestCacheConfiguration;
+
+import java.util.Arrays;
+import java.util.List;
+
+import picocli.CommandLine;
+
+public class ContractCacheOptions implements CLIOptions<JumpDestCacheConfiguration> {
+
+  public static final String CONTRACT_CACHE_WEIGHT = "--Xcontract-code-cache-weight-kb";
```

Comment:
The internals of this cache have gotten more specific. Should we rename the option for accuracy, or would that be detail the user doesn't need?

Reply:
Would it be useful to scope the wider configuration as something broader ie `EvmConfiguration`?  And inside that have something specific with an evm prefix?

Reply:
Good ideas all. I'm renaming both config and options classes. New cli flag will be `--Xevm-jumpdest-cache-weight-kb`